components/tabs/tab.js

```diff
@@ -38,6 +41,14 @@ class Tab extends TabMixin(LitElement) {
 		return styles;
 	}
 
+	updated(changedProperties) {
+		super.updated(changedProperties);
+
+		if (changedProperties.has('text')) {
```

Comment:
Do we want the event dispatched during initializing/first update? If not, we might need to check that the old value isn't `undefined`.

Reply:
I think we would - perhaps the content is delayed for some reason and so the slotchange stuff has happened but the text comes in late. This would be consistent with how the current `d2l-tab-panel-text-changed` event behaves.

Reply:
I guess it's not surprising that `render` is called less with the new code, since it's no longer rendering the tab handles internally.